discovery-handlers/debug-echo/src/discovery_handler.rs

```diff
@@ -76,53 +90,56 @@ impl DiscoveryHandler for DiscoveryHandlerImpl {
                     break;
                 }
 
+                //Standardize the procedures as same as the method used in OPC protocol discovery handler
```

Comment:
should this comment and standardization move to the `akri-shared` crate?

Reply:
Noted, thanks.